proxy/src/source.rs

```diff
@@ -77,13 +81,13 @@ impl From<radicle_registry_client::Project> for Project {
             })
             .collect();
 
-        Self {
-            id: p.id.clone().into(),
+        Ok(Self {
+            id: p.id.clone().try_into().unwrap(),
```

Comment:
Again, finding it weird that we call a partial function, `unwrap`, inside something that returns a `Result`.